scripts/parse_repo_json.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import json
+
+
+
+
+repo_json = json.loads(str(sys.stdin.readline()))
+
+out_string = ''
+for repo in repo_json['data']:
+    if repo['type'] == 'project':
+        repo_name = repo['id'].replace(' ','')
+        out_string += repo_name+' '
+
+print (out_string)
```

Comment:
Consider adding `strip()` to remove a trailing space, or avoid adding an extra space. 
```suggestion
print(out_string.strip())
```